### ISSUE TITLE
Trousseau sign

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10607]
**Preferred term label:**

**Synonyms**
carpal spasm

**Definition (free text, please give PubMed ID)**
PMID: [28722928] Trousseau sign: After inflating a blood pressure cuff from 10 mm Hg to 20 mm Hg over the patient's systolic blood pressure, the Trousseau sign is considered positive if carpal spasm develops, with flexion of the thumb and adduction of the finger within 3 minutes. However, this can be painful, so the time should be noted, and the cuff should be deflated once the test is positive.

**Parent term (use hpo.jax.org/app)**
Muscle spasm HP:0003394

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Secondary hypoparathyroidism due to impaired parathormon secretion ORPHA:140286

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
added as 
Trousseau sign
HP:6000919
